pony/orm/tests/test_decompiler.py

```diff
@@ -93,7 +96,85 @@ def get_condition_values(cond):
 
 
 class TestDecompiler(unittest.TestCase):
-    pass
+    def assertDecompilesTo(self, src, expected):
+        # skip test due to ast.dump has no indent parameter
+        if sys.version_info[:2] <= (3, 8):
+            return
+
+        code = compile(src, '<?>', 'eval').co_consts[0]
+        import dis
+        print(dis.dis(code))
+        dc = Decompiler(code)
+        expected = textwrap.dedent(expected).strip()
+        self.maxDiff = None
+        self.assertMultiLineEqual(expected, ast.dump(dc.ast, indent=2))
+
```

Comment:
I'm suggesting a not-so-better workaround:

```suggestion
        self.assertMultiLineEqual(self._strip_each_line(expected), ast.dump(dc.ast, indent=2))

    def _strip_each_line(self, string: str) -> str:
        return ' '.join(
            l.strip() for l in string.splitlines()
        ).replace('( ', '(').replace('[ ', '[').strip()
```

As far as I tested, it looks like it is working.